keras_cv/layers/regularization/drop_path.py

```diff
@@ -50,7 +50,11 @@ class DropPath(keras.layers.Layer):
     def __init__(self, rate=0.5, seed=None, **kwargs):
         super().__init__(**kwargs)
         self.rate = rate
-        self.seed = keras.random.SeedGenerator(seed=seed)
+        self._seed_val = seed
+        if not isinstance(seed, random.SeedGenerator):
```

Comment:
The seed value that is expected for drop_path is an `integer`. This check is unnecessary, we should directly initialize the `SeedGenerator` with the seed.

Reply:
Done